juneau-core/src/main/java/org/apache/juneau/internal/StringUtils.java

```diff
@@ -618,7 +618,9 @@ public static boolean isEmpty(String s) {
 	 * @return <jk>true</jk> if specified string is <jk>null</jk> or it's {@link #toString()} method returns an empty string.
 	 */
 	public static boolean isEmpty(Object s) {
-		return s == null || s.toString().isEmpty();
+		if( s == null ) return true;
+		if( s instanceof List ) return ((List) s).size() == 0;

```

Comment:
Of course, I missed the semantic that you want to return true if s is null. So you need the null check in addition to the equals or isEmpty check.

Too much javascript for me I guess. :(